### PR TITLE
Expand first sentence on fixtures into a paragraph

### PR DESCRIPTION
**Pull request by kpinc**
I would like to hyperlink just the first word, "Fixtures", but I don't think this is possible with RST  given that there is a pre-existing link target called "fixtures".

### CODE REVIEW

**Comment by blueyed** [comment]
> I would like to hyperlink just the first word, "Fixtures", but I don't think this is possible with RST given that there is a pre-existing link target called "fixtures".

You could rephrase it maybe into "<Fixtures in software tests>_ are used for initialisation." (or similar), which also makes it more obvious that this is not about pytest's fixtures in particular.

**Comment by kpinc** [comment]
On Sat, 15 Feb 2020 06:28:23 -0800
Daniel Hahler <notifications@github.com> wrote:

> > I would like to hyperlink just the first word, "Fixtures", but I
> > don't think this is possible with RST given that there is a
> > pre-existing link target called "fixtures".  
> 
> You could rephrase it maybe into "<Fixtures in software tests>_ are
> used for initialisation." (or similar), which also makes it more
> obvious that this is not about pytest's fixtures in particular.

Done.  I like the clarity about the link not having
to do with pytest in particular.  See if you like the
wording.

Regards,

Karl <kop@karlpinc.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

File: doc/en/fixture.rst [hunk 1]
```diff
@@ -10,15 +10,16 @@ pytest fixtures: explicit, modular, scalable
...
-so that tests execute reliably and produce consistent, repeatable,
-results.  Initialization may setup services, state, or other operating
-environments.  These are accessed by test functions though arguments;
-for each fixture used by a test function there is typically a
-parameter (named after the fixture) in the test function's definition.
+`Software test fixtures`_ initialize test functions.  They provide a
+fixed baseline so that tests execute reliably and produce consistent,
+repeatable, results.  Initialization may setup services, state, or
+other operating environments.  These are accessed by test functions
+though arguments; for each fixture used by a test function there is
```
Comment (blueyed) [inline doc/en/fixture.rst]:
```suggestion
through arguments; for each fixture used by a test function there is
```

**Comment by kpinc** [comment]
On Sat, 15 Feb 2020 06:53:54 -0800
Daniel Hahler <notifications@github.com> wrote:

> There's a typo though.

Thanks.  Fixed.

Regards,

Karl <kop@karlpinc.com>
Free Software:  "You don't pay back, you pay forward."
                 -- Robert A. Heinlein

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**